src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/wwwroot/css/app.css

```diff
@@ -62,3 +62,25 @@ a, .btn-link {
     .blazor-error-boundary::after {
         content: "An error has occurred."
     }
+
+#blazor-default-loading svg {
+    width: 120px;
+    height: 120px;
+    stroke-width: 8px;
+    margin-left: auto;
+    margin-right: auto;
+    display: block;
+}
+
+#blazor-default-loading circle {
+    fill: none;
+    stroke: #e0e0e0;
+    transform-origin: 50% 50%;
+    transform: rotate(-90deg);
+}
+
+#blazor-default-loading #blazor-default-loading-progress {
+    stroke: #1b6ec2;
+    transition: 0.50s stroke-dashoffset ease-in-out;
+    display: none;
+}
```

Comment:
Should we add some styling for `blazor-default-loading-percentage` (font family, size, color, weight, etc.)?